### PR TITLE
Consistent imports

### PR DESCRIPTION
**Pull request by eumiro**
This is the follow-up of #864 consolidating all `import` statements across the repo.

First, all unused imports are deleted, then the Python2 `__future__` imports and then the rest to have the same names in the whole codebase.

When this is approved, I'd squash all commits within this PR to a single one.

There's one big chapter missing: `import tables as tb` (it's my proposal to use the `tb` alias). But that would be a bigger separate PR.

### CODE REVIEW

File: tables/tests/test_attributes.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """This test unit checks node attributes that are persistent (AttributeSet)."""
 
+import datetime as dt
```
Comment (avalentino) [inline tables/tests/test_attributes.py:3]: It is my personal taste but I would prefer to keep the original name in this case

File: tables/tests/test_enum.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 """Test module for enumerated types under PyTables."""
 
 import operator
-import itertools
+import itertools as it
```
Comment (avalentino) [inline tables/tests/test_enum.py:14]: also in this case I would prefer to keep "itertools"

**Review by avalentino — CHANGES_REQUESTED** [review]

File: tables/tests/test_attributes.py (hunk 1 above)
Comment (eumiro) [reply tables/tests/test_attributes.py]: No problem, changed.

File: tables/tests/test_enum.py (hunk 2 above)
Comment (eumiro) [reply tables/tests/test_enum.py]: There are only a few lines of code, no problem.

**Review by avalentino — APPROVED** [review]

**Pull request merged**